README.md

```diff
@@ -1,36 +1,31 @@
-[![Manager](https://github.com/thoughtworks/maeve-csms/workflows/Manager/badge.svg)](https://github.com/thoughtworks/maeve-csms/actions/workflows/manager.yml)
-[![Gateway](https://github.com/thoughtworks/maeve-csms/workflows/Gateway/badge.svg)](https://github.com/thoughtworks/maeve-csms/actions/workflows/gateway.yml)
+[![Manager](https://github.com/thoughtworks/Maeve-csms/workflows/Manager/badge.svg)](https://github.com/thoughtworks/Maeve-csms/actions/workflows/manager.yml)
+[![Gateway](https://github.com/thoughtworks/Maeve-csms/workflows/Gateway/badge.svg)](https://github.com/thoughtworks/Maeve-csms/actions/workflows/gateway.yml)
 
 # MaEVe
 
-MaEVe is an EV charge station management system (CSMS). It began life as a simple proof of concept for
-implementing ISO-15118-2 Plug and Charge (PnC) functionality and remains a work in progress. It is hoped that over
-time it will become more complete, but already provides a useful basis for experimentation.
+Maeve is an EV Charge Station Management System (CSMS). It began life as a simple proof of concept for
+implementing ISO-15118-2 Plug and Charge (PnC) functionality and remains a work in progress. Over
+time it will become more complete. It provides a useful basis for experimentation with [Everest](https://github.com/EVerest).
 
-The system currently integrates with [Hubject](https://hubject.stoplight.io/) for PnC functionality.
```

Comment:
This seems like an important piece of information that isn't contained elsewhere.

Reply:
I rewrote it as "Maeve integrates with [Hubject](https://hubject.stoplight.io/) for Plug and Charge (PnC) functionality."